share/spice/forecast/forecast.css

```diff
@@ -274,3 +272,20 @@
 #fe_temp_switch .is-active {
 	font-weight: bold;
 }
+
+/*
+    Dark mode specific changes
+*/
+
+.dark-bg .fe_label a:hover {
+    color: #dadada;
+    text-decoration: underline;
+}
+
+.dark-bg a#fe_temp_switch:hover {
+    color: #c2c2c2;
+}
+
+.dark-bg .fe_icon { display: none; }
```

Comment:
Ah I see you're moving these. The `.dark-header` class is still present on the page in the dark theme -- this does't seem necessary. Please remove this